vnext/ReactUWP/Views/FrameworkElementViewManager.cpp

```diff
@@ -335,7 +335,7 @@ void FrameworkElementViewManager::UpdateProperties(
         AnnounceLiveRegionChangedIfNeeded(element);
       } else if (propertyName == "accessibilityPosInSet") {
         if (propertyValue.isNumber()) {
-          auto value = static_cast<int>(propertyValue.getInt());
+          auto value = static_cast<int>(propertyValue.asDouble());
```

Comment:
>auto [](start = 10, length = 4)

Not directly related to your CR, but do you know why we have properties for "position in set" and "size", but not "level"? These 3 go together to express a list and the indentation level of the list members.

Reply:
We plan on pushing these properties back up into RN, so we followed the aria specifications to be more cross-platform and less windows-specific, which only includes posInSet and setSize.

Reply:
There is an aria-level property that exists. Doesn't it map to the level property in UIA?